ambari-web/app/controllers/wizard.js

```diff
@@ -875,7 +875,10 @@ App.WizardController = Em.Controller.extend(App.LocalStorage, App.ThemesMappingM
     var interval = setInterval(function () {
       if (miscController.get('dataIsLoaded')) {
         if (self.get("content.hdfsUser")) {
-          self.set('content.hdfsUser', miscController.get('content.hdfsUser'));
+          var hdfsUser = miscController.get('users').filterProperty('filename', 'hadoop-env.xml').findProperty('name','hdfs_user');
```

Comment:
It doesn't extract the actual value of user_group, so it stays hardcoded.
When you refresh on Manual commands step sometimes loading goes in a different order and UI uses the default(hardcoded) value of hdfsUser.